src/buildblock/findSTIRConfig.cxx

```diff
@@ -0,0 +1,24 @@
+
+#include <cstdlib>
+#include "stir/findSTIRConfig.h"
+#include "stir/info.h"
+#include <iostream>
+#include <fstream>
+
+START_NAMESPACE_STIR
+
+std::string find_STIR_config_file(std::string filename){
+    std::string dir= getenv("STIR_CONFIG_DIR");
```

Comment:
getenv() returns a NULL pointer. I tried changing variable and ONLY "PATH" returns a value. any suggestion?

Reply:
on Linux/Unix? In `bash`, use `export STIR_CONFIG_DIR`. with `csh` use `setenv`

Reply:
OK, so the plan is to export this via bash script? I thought that when we set these variable via Cmake they will be usable in the code

Reply:
ok. we can add an option to CMake. i guess we said that. This would replace the `CMAKE_INSTALL_PREFIX` stuff below. However, it cannot use `getenv`. CMake doesn't set your environment variables (and if it did, it would only be during building). 

So, yes. let's add a CMake cached variable `STIR_CONFIG_DIR` defaulting to `${CMAKE_INSTALL_PREFIX}/share/stir/config`, add a corresponding `#cmakedefine` line in https://github.com/UCL/STIR/blob/master/src/cmake/STIRConfig.h.in, and replace the lines below to say
`dir = STIR_CONFIG_DIR`. The env variable can then be used to override that value.